### PR TITLE
Presentation: Can't remove related properties from supplemental rulesets

### PR DESCRIPTION
**Pull request by Alina657**
Documentation for fixed bugs.
https://dev.azure.com/bentleycs/iModelTechnologies/_git/imodel02/pullrequest/244298

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
     alert("Integrated graphics are in use. If a discrete GPU is available, consider switching your device or browser to use it.");
 ```
 
+### Fixed inconsistent property representation in a property grid
+
+Previously when using `nestedRelatedProperties` attribute, nested related properties were merged or not merged depending on whether at least one property of an intermediate class was selected. Now in both cases properties are not merged.
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:100]:
```suggestion
Previously, when using [RelatedPropertiesSpecification.nestedRelatedProperties]($presentation-common) attribute, the properties were loaded differently based on whether on whether parent specification included any properties or not. Now the behavior is consistent.
```

File: docs/changehistory/NextVersion.md [hunk 2]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
...
+
+```json
+{
+  "id": "example",
+  "rules": [
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:105]: To avoid making the example so massive, could you format the rulesets in a way to save vertical space? E.g.:
```json
"rules": [{
  "ruleType": "Content",
  "specifications": [{
    "specType": "ContentInstancesOfSpecificClasses",
    "classes": { "schemaName": "BisCore", "classNames": ["GeometricModel3d"], "arePolymorphic": true }
  }]
}, {
  ...
```

File: docs/changehistory/NextVersion.md [hunk 3]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
...
+              "propertiesSource": {
+                "relationship": {
+                  "schemaName": "BisCore",
+                  "className": "GeometricElement3dIsInCategory"
+                },
+                "direction": "Forward",
+                "targetClass": {
+                  "schemaName": "BisCore",
+                  "className": "Category"
+                }
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:150]: I don't think we need `targetClass` in this example.

File: docs/changehistory/NextVersion.md [hunk 4]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
...
+              },
+              "properties": "*",
+              "handleTargetClassPolymorphically": true
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:153]: I don't think we need `properties` or `handleTargetClassPolymorphically` here.

File: docs/changehistory/NextVersion.md [hunk 5]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
...
+          ],
+          "handleTargetClassPolymorphically": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+|                               | before                                                                                                                            | after                                                                                                                             |
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:164]: Let's add this to the first column header: **Value of `"properties"` attribute**.

And then change values for the column to `"_none_"` and `["UserLabel"]`.

File: docs/changehistory/NextVersion.md [hunk 6]
```diff
@@ -95,6 +95,166 @@ iTwin.js applications can now check [WebGLRenderCompatibilityInfo.usingIntegrate
...
+      "properties": [
+        {
+          "name": "Identifier",
+          "isDisplayed": false
+        }
+      ]
+    }
+  ]
+}
+```
```
Comment (grigasp) [inline docs/changehistory/NextVersion.md:256]: I'd reword to this:

For example, the [default BisCore supplemental ruleset](https://github.com/iTwin/itwinjs-core/blob/504a7b88e9ade29cc306bd55d093be1d76ddad43/presentation/backend/assets/supplemental-presentation-rules/BisCore.PresentationRuleSet.json#L372-L419) automatically includes `ExternalSourceAspect.Identifier` property to all Elements' content when they have such aspect. It's now possible to disable that rule using one of these approaches:

- Set `properties` to `"_none_"` or `[]`:
  
  ```json
  {
    "ruleType": "ContentModifier",
    "class": { "schemaName": "BisCore", "className": "Element" },
    "relatedProperties": [{
      "propertiesSource": {
        "relationship": { "schemaName": "BisCore", "className": "ElementOwnsMultiAspects" },
        "direction": "Forward",
        "targetClass": { "schemaName": "BisCore", "className": "ExternalSourceAspect" }
      },
      "properties": "_none_"
    }]
  }
  ```

- Disable property display:

  ```json
  {
    "ruleType": "ContentModifier",
    "class": { "schemaName": "BisCore", "className": "Element" },
    "relatedProperties": [{
      "propertiesSource": {
        "relationship": { "schemaName": "BisCore", "className": "ElementOwnsMultiAspects" },
        "direction": "Forward",
        "targetClass": { "schemaName": "BisCore", "className": "ExternalSourceAspect" }
      },
      "properties": [{
        "name": "Identifier",
        "isDisplayed": false
      }]
    }]
  }
  ```

**Review by grigasp — CHANGES_REQUESTED** [review]
The screenshots shouldn't include our test-app specific stuff (e.g. the **Copy** button).

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (saskliutas) [reply docs/changehistory/NextVersion.md:100]: Typo in suggested change `... on whether on whether ...`

**Review by grigasp — APPROVED** [review]
